docs/design/coreclr/profiling/davbr-blog-archive/Debugging - SOS and IDs.md

```diff
@@ -9,7 +9,7 @@ SOS.DLL is a debugger extension DLL that ships with the CLR.  You'll find it sit
 
 In windbg, you'll need mscorwks.dll to load first, and then you can load SOS.  Often, I don't need SOS until well into my debugging session, at which point mscorwks.dll has already been loaded anyway.  However, there are some cases where you'd like SOS loaded at the first possible moment, so you can use some of its commands early (like !bpmd to set a breakpoint on a managed method).  So a surefire way to get SOS loaded ASAP is to have the debugger break when mscorwks gets loaded (e.g., "sxe ld mscorwks").  Once mscorwks is loaded, you can load SOS using the .loadby command:
 
-| 
+|
 ```
 0:000\> **sxe ld mscorwks**
```

Comment:
Same comment as before, about the asterisks and the vertical bar.

Reply:
Same as above.